tests/e2e/lib.sh

```diff
@@ -219,37 +218,25 @@ patch_resources_for_test() {
 
     kubectl -n stackrox patch svc central-loadbalancer --patch "$(cat "$TEST_ROOT"/tests/e2e/yaml/endpoints-test-lb-patch.yaml)"
     kubectl -n stackrox apply -f "$TEST_ROOT/tests/e2e/yaml/endpoints-test-netpol.yaml"
+
+    for target_port in 8080 8081 8082 8443 8444 8445 8446 8447 8448; do
+        check_endpoint_availability "$target_port"
+    done
+}
+
+check_endpoint_availability() {
+    local target_port="$1"
     # shellcheck disable=SC2034
     for i in $(seq 1 20); do
-        if curl -sk --fail "https://${API_HOSTNAME}:8446/v1/metadata" &>/dev/null; then
+        if echo "Endpoint check" 2>/dev/null > /dev/tcp/"${API_HOSTNAME}"/"${target_port}"; then
```

Comment:
I vaguely remember some distros disable this `/dev/tcp` functionality in bash, so this might become a portability problem, but we can always just revert, right? :wink: